common/buildings/01_industry.txt

```diff
@@ -18,7 +18,16 @@ building_food_industry = {
 		pmg_ownership_food
 	}
 	required_construction = construction_cost_high
-
+	possible = {
+		error_check = {
+			severity = fail
+			this = {
+				owner = { 
+					NOT = { has_law = law_type:law_industry_banned } 
+				}
```

Comment:
```suggestion
	possible = {
	    owner = {
           not = {
               has_law = law_type:law_industry_banned
           }
       }  
	}
```

We don't need to be error checking all of these, it's just a waste of processing power. Simply setting possible to weather or not they have the law is enough.